client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -201,8 +202,18 @@ function SupportText( { hasPlanInCart, isJetpackNotAtomic } ) {
 	return <span>{ translate( 'Email support' ) }</span>;
 }
 
-function CheckoutSummaryFeaturesListDomainItem( { domain, hasMonthlyPlan, nextDomainIsFree } ) {
+function CheckoutSummaryFeaturesListDomainItem( {
+	domain,
+	hasMonthlyPlan,
+	hasPlanInCart,
+	nextDomainIsFree,
+} ) {
 	const translate = useTranslate();
+
+	if ( ! hasPlanInCart ) {
```

Comment:
Now that I look through this code some more, I wonder if it should be added to the `if ( domain.is_bundled ) {` check instead. Previously, buying a single domain with no credits would have shown the domain on its own with a check mark next to it.